package.json

```diff
@@ -145,6 +145,7 @@
     "fast-levenshtein": "^2.0.6",
     "file-loader": "^1.1.11",
     "fuse.js": "^3.2.0",
+    "gaba": "^1.0.0-beta.45",
```

Comment:
Echoing [what Whymarrh said in the other gaba PR](https://github.com/MetaMask/metamask-extension/pull/5844#discussion_r):

> We should pin this exactly so that we're not _automatically_ pulling in new prereleases